h/templates/app.html.jinja2

```diff
@@ -34,20 +34,6 @@
     {% for url in app_js_urls %}
     <script src="{{ url }}"></script>
     {% endfor %}
-
-    <!-- Analytics !-->
-    {% if ga_tracking_id %}
```

Comment:
If we are removing this then we can also remove the `ga_tracking_id` argument to `render_app_html`.